pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -574,8 +575,14 @@ public static void addColumnMetadataInfo(PropertiesConfiguration properties, Str
     if (defaultNullValue == null) {
```

Comment:
I don't think defaultNullValue can ever be null

Reply:
Correct, we have default null value for all kinds of types in FieldSpec. 
Removed.